### ISSUE TITLE
G0 - Doctype and lang issues

### ISSUE THREAD

**Issue by jpinsent** [brharp/hjckrrh#76]
Running a basic page through the w3c markup validation checker indicates that the wrong doctype is being used. @mmafe stated that the doctype is specified as xhtml when HTML5 markup is being used.

We are also missing the lang attribute for xhtml, however, this may be irrelevant after switching the doctype to HTML5

http://validator.w3.org/check?uri=https%3A%2F%2Fdev.web.uoguelph.ca%2Faodademo%2Fimages&charset=%28detect+automatically%29&doctype=Inline&group=0

**Comment by brharp** [comment]
Doctype is now "html" and lang set to "en"
